notebookUtils.py

```diff
@@ -157,9 +157,16 @@ def scriptPostSave(model, os_path, contents_manager, **kwargs):
 
     sandbox_id = os.environ['SANDBOX_ID']
     token = getStorageTokenFromEnv(log)
-    saveFile(os_path, sandbox_id, token, log)
     updateApiTimestamp(sandbox_id, token, log)
-    saveFolder('/data/.git', sandbox_id, token, log)
+
+    if 'HAS_PERSISTENT_STORAGE' in os.environ:
+        has_persistent_storage = os.environ['HAS_PERSISTENT_STORAGE'] == '1'
+    else:
+        has_persistent_storage = False
```

Comment:
This is a bit strange to me.  I think we can maybe use something like 
`has_persistent_storage = os.getenv('HAS_PERSISTENT_STORAGE', 'False').lower() in ('true', '1')` 
instead.  so it's more explicitly truthy/falsy